pkg/fleet/internal/exec/installer_exec.go

```diff
@@ -65,7 +69,7 @@ func (i *InstallerExec) newInstallerCmd(ctx context.Context, command string, arg
 		fmt.Sprintf("DD_SITE=%s", i.site),
 	}...)
 	cmd.Cancel = func() error {
-		return cmd.Process.Signal(os.Interrupt)
+		return nil
```

Comment:
mostly cosmetic so the error can actually be context canceled and not a signal send error